db/sql/dbsetup.sql

```diff
@@ -319,7 +324,8 @@ RIGHT JOIN fhir_endpoints AS endpts ON links.url = endpts.url
 LEFT JOIN fhir_endpoints_info AS endpts_info ON endpts.url = endpts_info.url
 LEFT JOIN fhir_endpoints_metadata AS endpts_metadata ON endpts_info.metadata_id = endpts_metadata.id
 LEFT JOIN vendors ON endpts_info.vendor_id = vendors.id
-LEFT JOIN npi_organizations AS orgs ON links.organization_npi_id = orgs.npi_id;
+LEFT JOIN npi_organizations AS orgs ON links.organization_npi_id = orgs.npi_id
+WHERE endpts_info.requested_fhir_version = '';
 
 CREATE INDEX fhir_endpoints_url_idx ON fhir_endpoints (url);
 CREATE INDEX fhir_endpoints_info_url_idx ON fhir_endpoints_info (url);
```

Comment:
On line 343 (can't click on the actual line), we could delete the fhirVersion index since it won't be used anymore, but I guess it also doesn't hurt to keep it?

Reply:
Probably makes more sense to get rid of it since we don't use it anywhere anymore so it would just be taking up space. Do you think an index should be added for the requested version and capability version fields?

Reply:
can't hurt!